tests/run_system_tests.py

```diff
@@ -42,5 +42,3 @@
     all_tests.addTest(tests)
 result = unittest.TextTestRunner(verbosity=2).run(all_tests)
 sys.exit(not result.wasSuccessful())
-
-sys.argv = ['unittest', '-v'] + tests
```

Comment:
This looks like something @alanconway left there by mistake. It is a dead code anyways. Deleting.